src/components/entropy/index.js

```diff
@@ -177,7 +178,21 @@ class Entropy extends React.Component {
       </div>
     );
   }
+  requestRecalculationOfBars() {
+    /* The <Entropy> component is now responsible for requesting a recalculation of the underlying entropy data
+    as this allows us to skip calculations when (e.g.) the panel is not rendered */
```

Comment:
_non-blocking_

Hmm, this feels like an anti-pattern to me. I would expect the `TOGGLE_PANEL_DISPLAY` action to trigger the entropy re-calculation or clear the entropy data. Then there's a clear connection that the toggling of the panel display will trigger re-calculation.

Reply:
Agreed. I moved to this style both here and in the related comment. Thanks!